app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDeclarationDetailsController.scala

```diff
@@ -426,21 +425,6 @@ class EnterDeclarationDetailsController @Inject() (
 
 object EnterDeclarationDetailsController {
 
-  final case class EntryDeclarationDetails(
-    dateOfImport: DateOfImport,
-    placeOfImport: String,
-    importerName: String,
-    importerEmailAddress: Email,
-    importerPhoneNumber: PhoneNumber,
-    declarantName: String,
-    declarantEmailAddress: Email,
-    declarantPhoneNumber: PhoneNumber
-  )
-
-  object EntryDeclarationDetails {
-    implicit val format: OFormat[EntryDeclarationDetails] = Json.format[EntryDeclarationDetails]
-  }
-
   val entryDeclarationDetailsForm: Form[EntryDeclarationDetails] = Form(
```

Comment:
if you are removing Entry Number related code then you might as well delete the the Form definition as well.